src/storage.rs

```diff
@@ -65,8 +67,8 @@ impl Storer for RedactStorer {
         }
     }
 
-    async fn get_collection(&self, path: &str, skip: i64, page_size: i64) -> Result<DataCollection, Rejection> {
-        match reqwest::get(&format!("{}/data/{}?skip={}&page_size={}", self.url, path, skip, page_size)).await {
+    async fn get_collection(&self, path: &str, skip: i64) -> Result<DataCollection, Rejection> {
+        match reqwest::get(&format!("{}/data/{}?skip={}&page_size={}", self.url, path, skip, self.collection_page_size)).await {
```

Comment:
We need to do a review of all of our user-accepted parameters and how we're using them. We're not doing input sanitization anywhere and it could potentially be used maliciously.